server.js

```diff
@@ -68,6 +68,9 @@ const hbs = create({
     feedLink() {
```

Comment:
since you're adding the `feedUrl()`, would you mind refactoring the place in `main.hbs` that calls this, put the `<link>` tag directly into the template, and just call the same `feedUrl` helper for the `href` attribute? Then this helper can be deleted entirely.